content/case-studies/persana-case-study.md

```diff
@@ -0,0 +1,213 @@
+---
+date: 2024-07-26T12:00:00Z
+title: "Persana Case Study: Lead Generation AI Platform"
+heroImage: images/case-studies/persana.png 
+overview: "Persana is an AI-driven sales prospecting tool designed to find, enrich, and personalize outbound leads at scale. It helps users, such as sales teams and marketers, save time by building qualified lead lists, creating personalized messages, and increasing response rates. Key features include lead enrichment, AI-generated messaging, LinkedIn prospecting via a Chrome extension, and automated workflows. Users can access pre-built lead lists, import their own, and utilize various enrichments to gain deeper insights into leads and companies."
+domain: "Artificial Intelligence (AI)"
+url: "/projects/case-studies/persana-case-study"
+
+---
+
+## __Technologies and Technical Specifications__
+
+{{< technologies >}}
+
+<img src="/images/case-studies/react.png" alt="React JS"/>
+<img src="/images/case-studies/tailwind.png" alt="Tailwind CSS"/>
+<img src="/images/case-studies/vite.png" alt="Vite"/>
+<img src="/images/case-studies/nest.png" alt="Nest JS"/>
+<img src="/images/case-studies/mongodb.png" alt="MongoDB"/>
+<img src="/images/case-studies/redis.png" alt="Redis"/>
+<img src="/images/case-studies/vue.png" alt="Vue JS"/>
+<img src="/images/case-studies/framer.png" alt="Framer"/>
```

Comment:
In mobile view there must be only one logo in a row. Currently it's showing two. Please correct it.

Reply:
suggested change done in commit [9e1bead](https://github.com/scalereal/website/pull/160/commits/9e1beade7fa0bb2f8fbaccd8fb7cf6fc68e3076d)